File: 4.Clients/WebApi/ClientApp/src/entities/hr-stage.ts
import { StageStatusEnum } from './enums/stage-status.enum';
import { EnglishLevelEnum } from './enums/english-level.enum';
import { Stage } from 'src/entities/stage';
import { RejectionReasonsHrEnum } from './enums/rejection-reasons-hr.enum';

export class HrStage extends Stage{
    actualSalary: number;
    wantedSalary: number;
    englishLevel: EnglishLevelEnum;
    rejectionReasonHR?: RejectionReasonsHrEnum;
}

File: 1.Domain/Domain.Model/HrStage.cs
using Domain.Model.Enum;
using System;
using System.Collections.Generic;
using System.Text;

namespace Domain.Model
{
    public class HrStage: Stage
    {
        public float ActualSalary { get; set; }
        public float WantedSalary { get; set; }
        public EnglishLevel EnglishLevel { get; set; }
        public RejectionReasonsHr RejectionReasonsHr { get; set; }
    }
}
